### PR TITLE
kubeadm: add new test job for super-admin.conf

### PR DESCRIPTION
**Pull request by neolit123**
xref https://github.com/kubernetes/kubeadm/issues/2414
/hold for https://github.com/kubernetes/kubeadm/pull/2949

### CODE REVIEW

File: config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder-discovery.yaml [hunk 1]
```diff
@@ -46,7 +46,7 @@ periodics:
           cpu: 2000m
 
 - name: ci-kubernetes-e2e-kubeadm-kinder-discovery-1-28
-  cluster: eks-prow-build-cluster
+  cluster: k8s-infra-prow-build
```
Comment (neolit123) [inline config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder-discovery.yaml:49]: these files are autogenerated, from config and temaplates.
adding manually the EKS cluster will not work.

this in particular is left over when we tried to migrate the kubeadm jobs to EKS.

in any, case if we try to do it again we have to modify the config/templates.
push PRs, with the generated diffs in k/kubeadm and k/test-infra.

**Review by neolit123 — COMMENTED** [review]
/cc @chendave @SataQiu

**Review by SataQiu — COMMENTED** [review]
/lgtm

File: config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder-discovery.yaml (hunk 1 above)
Comment (chendave) [reply config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder-discovery.yaml:49]: sgtm!

Comment (chendave) [reply config/jobs/kubernetes/sig-cluster-lifecycle/kubeadm-kinder-discovery.yaml:49]: /lgtm

**Pull request merged**